src/pages/ProductsPage/ProductCard/styles.ts

```diff
@@ -0,0 +1,81 @@
+import { SxProps } from '@mui/material';
+
+export const mainCardStyle: SxProps = {
+  maxWidth: 400,
+};
+
+export const cardMediaContainer: SxProps = {
+  position: 'relative',
+};
+
+export const cardMediaChildContainer: SxProps = {
+  position: 'absolute',
+  padding: '10px',
+};
+
+export const productNameTypography: SxProps = {
+  fontWeight: 'bolder',
+};
+
+export const productSaleAvatar: SxProps = {
+  bgcolor: 'white',
+  color: 'black',
+  width: 45,
+  height: 45,
+  border: 'solid black',
+};
+
+export const productPriceItem: SxProps = {
+  fontStyle: 'italic',
+  fontWeight: 'bold',
+  fontSize: '25px',
+};
+
+export const productCardBadgeBox: SxProps = {
```

Comment:
Do not use the Box ending if it is not a real box wish contain a lot of items

Reply:
This style has been renamed